requirements.txt

```diff
@@ -12,3 +12,9 @@ markdown==3.5
 PyPDF2==3.0.1
 gitpython==3.1.43
 beautifulsoup4==4.12.3
+#If use PyTables
```

Comment:
I don't see PyTables, tables used below.  Can't we eliminate these new requirements?